src/net/tcp/connection.cpp

```diff
@@ -649,6 +653,40 @@ void Connection::rtx_ack(const seq_t ack) {
   //  x-rtx_q.size(), rtx_q.size());
 }
 
+uint32_t Connection::calculate_rcv_wnd() const
```

Comment:
Could be inlined ?

Reply:
Inlining it means rebuilding half the OS when tampering with it. I wouldn't say the implementation is 100% yet.
Also, won't the compiler inline it for us inside?